server.js

```diff
@@ -22,7 +22,12 @@ const server = new ApolloServer({
     const { token } = req.headers || '';
     if (token) {
       const user = verifyUser(token);
-      if (!user) throw new AuthenticationError('Invalid authorization token');
+      if (!user) {
+        return {
+          statusCode: 401,
+          message: 'Invalid authorization token',
```

Comment:
Message should be `INVALID_AUTHORIZATION_TOKEN`

Reply:
we have go INVALID_PERMISSION, ill stick to it